### PR TITLE
new button added by Arvind0302

### PR DESCRIPTION
**Pull request by Arvind0302**
## Have you read the Contribution Guidelines? If not, check [README.md](https://github.com/Design-and-Code/css-buttons#some-rules-to-contribute) file first.

YES.

## Description

I had added a new button with some cool hovering effect. Please merge my pr and add the hacktoberfest accepted label on it. Thank you!

## Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.

> mark `[X]` to make it checked.

## Related Issues

Fixes Issue # (write number after `#`)

### CODE REVIEW

**Comment by Arvind0302** [comment]
Screenshot (85)

File: buttons/buttons.css [hunk 1]
```diff
@@ -2179,3 +2179,29 @@ a:focus-visible {
...
 
+/*Arvind0302 button start */
+.arvind0302-btn-1
+{
+    position: relative;
+    text-align: center;
+    width: 200px;
+    padding: 20px;
+    font-size: 15px;
+    color: #15f4ee;
```
[the remark this answers is not in the document]
Comment (Arvind0302) [reply buttons/buttons.css:2190]: I didn't understand,  did u mean to say that i should change the text color before hovering?

[the remark this answers is not in the document]
Comment (Rajkumar-justcoder) [reply buttons/buttons.css:2190]: yup that a default state of text before hover

[the remark this answers is not in the document]
Comment (Arvind0302) [reply buttons/buttons.css:2190]: Ok, i will do that..

**Review by Rajkumar-justcoder — COMMENTED** [review]
other users buttons css are removed 
image
image

as this pr is now more mess up so closing this pr 
pls reopen pr and make push your code again and take care while resolving conflict

**Pull request closed without merging**